src/rpcrawtransaction.cpp

```diff
@@ -529,8 +541,18 @@ Value signrawtransaction(const Array& params, bool fHelp)
 
             "\nResult:\n"
             "{\n"
-            "  \"hex\": \"value\",   (string) The raw transaction with signature(s) (hex-encoded string)\n"
-            "  \"complete\": n       (numeric) if transaction has a complete set of signature (0 if not)\n"
+            "  \"hex\" : \"value\",   (string) The hex-encoded raw transaction with signature(s)\n"
+            "  \"complete\" : b,    (boolean) Whether the transaction has a complete set of signatures\n"
+            "  \"errors\" : [       (json array of objects) Script verification errors (if there are any)\n"
+            "    {\n"
+            "      \"txid\" : \"hash\",       (string) The transaction hash\n"
```

Comment:
Sorry if I wasn't clear. I just suggest making the help output a bit clearer. I understand what it does and why, but the help text doesn't explain _which_ txid is being reported.